src/PolicyInsights/PolicyInsights.Test/SessionRecords/Microsoft.Azure.Commands.PolicyInsights.Test.ScenarioTests.PolicyStateSummaryTests/ManagementGroupScope.json

```diff
@@ -1,8 +1,8 @@
 {
   "Entries": [
     {
-      "RequestUri": "/providers/Microsoft.Management/managementGroups/azgovtest4/providers/Microsoft.PolicyInsights/policyStates/latest/summarize?api-version=2018-04-04&$top=10&$from=2018-03-31T00:00:00Z",
-      "EncodedRequestUri": "L3Byb3ZpZGVycy9NaWNyb3NvZnQuTWFuYWdlbWVudC9tYW5hZ2VtZW50R3JvdXBzL2F6Z292dGVzdDQvcHJvdmlkZXJzL01pY3Jvc29mdC5Qb2xpY3lJbnNpZ2h0cy9wb2xpY3lTdGF0ZXMvbGF0ZXN0L3N1bW1hcml6ZT9hcGktdmVyc2lvbj0yMDE4LTA0LTA0JiR0b3A9MTAmJGZyb209MjAxOC0wMy0zMVQwMCUzQTAwJTNBMDBa",
```

Comment:
Please rerecord these test, we need rerecording for a new API version, not just manipulation of the json files.

Reply:
Done.